webhooks-extension/config/triggers_prototype/github/webhook.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: webhook-task
+  namespace: tekton-pipelines
+spec:
+  inputs:
+    params:
+    - name: Mode
+      description: Whether or not to create or delete the webhook
+      default: "create"
+    - name: ExternalUrl
+      description: The external access URL (used for creating the webhook)
```

Comment:
make this clearer i.e for example the ingress external URL in front of the EventListener